src/app/modules/reports/components/time-entries-table/time-entries-table.component.ts

```diff
@@ -28,11 +29,29 @@ export class TimeEntriesTableComponent implements OnInit, OnDestroy, AfterViewIn
       },
       {
         extend: 'excel',
+        exportOptions: {
+          format: {
+            body: (data, row, column, node) => {
+              return column === 3 ?
```

Comment:
We call it magic number. It's a code smell. Please give it a meaningful variable name